### PR TITLE
Increased waiting time to sign image for longer builds than 10 minute…

### PR DESCRIPTION
**Pull request by radoslawc**
…s, fixes for test-infra image signing

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

/kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
some images take longer to build than 10 minutes (kpt backstage plugins for example), fixes for test-infra images signing
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by efiacor** [comment]
Would it be worthwhile seeing if the Dockerfiles builds could be streamlined?

File: .prow.yaml [hunk 1]
```diff
@@ -310,7 +310,7 @@ postsubmits:
             - "-c"
             - |
               executor --context=${PWD}/images/releaser \
-                --dockerfile=Dockerfile --destination=nephio/releaser:${PULL_BASE_REF} \
+                --dockerfile=Dockerfile --destination=nephio/releaser:${BUILD_ID} \
```
Comment (vjayaramrh) [inline .prow.yaml:313]: Where is ${BUILD_ID} defined? Thanks

Comment (radoslawc) [reply .prow.yaml:313]: Those are Prow job env variables: https://docs.prow.k8s.io/docs/jobs/#job-environment-variables

Comment (vjayaramrh) [reply .prow.yaml:313]: Thanks

**Pull request merged**